DataConnectors/Gitlab/README.md

```diff
@@ -0,0 +1,19 @@
+## Description
+This log source uses logic app workflow with log analytics send data plugin. This Gitlab connector uses 3 types of audit logs (Project Audit, Group Audit and user events). For additional audit logs, you can modify the workflow with additional apis. For more information on Gitlab audit events, refer to https://docs.gitlab.com/ee/api/audit_events.html
+
+## Prerequisites
+1. Gitlab personal access token with read permissions for groups and projects.
+2. Gitlab Group ID and Project ID required for audit log ingestion.
+3. Log analytics workspace ID and shared Key
+
+## Logic APP Design
+![LogicAppWorkflow](./LogicApp.png)<br>
+
+
+## Deploy the Logic App template
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/<Tobeadded>)
```

Comment:
This one is tough.  It almost feels as though it should not be a link if the link is invalid.

Reply:
Commented that for now.